code/resources/css/style.css

```diff
@@ -0,0 +1,372 @@
+/*                    GLOBAL CONTAINER             */
+/* ----------------------------------------------- */
+
+/*Contains the sections: Header, Featured News, About Us, Footer*/
+
+.global-container {
+  padding: 3%;
+}
+
+/* ----------------------------------------------- */
+/*                REUSABLE COMPONENTS              */
+/* ----------------------------------------------- */
+
+/* ----- BUTTONS ----- */
+
+.cta-button {
+  background-color: #FC6C3C; 
+  color:#FCDDD2;
+  font-size: 1.1vw;
+  border-radius: 200px;
+  padding: 1.5%;
+  margin-top: 1%;
+}
+
+.cta-button:hover {
+  box-shadow: 8px 8px #B7448E;
+  transition: .3s;
+}
+
+/* ----------------------------------------------- */
+/*                    HEADINGS                     */
+/* ----------------------------------------------- */
+
+h1 {
+  font-family: 'Amaranth', 'Arial', cursive;
+  color: #FC9270;
+  font-size: 5vw;
+}
+
+h2 {
+  color: #FC6424;
+  font-size: 2vw;
+  margin-bottom: 1%;
+}
+
+h3 {
+  color: #FC6434;
+  font-size: 1.5vw;
+}
+
+.header-subtitle {
+  color:#FCDDD2;
+}
+
+
+
+/* ----------------------------------------------- */
+/*                    HEADER                       */
+/* ----------------------------------------------- */
+
+ .hero-container {
+  background-image: -webkit-linear-gradient(rgba(0, 0, 0, 0.7), rgba(0, 0, 0, 0.7)), url(img/hero-img.jpg);
+  background-size: cover;
+  background-position: center center;
+  background-attachment: fixed;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+ } 
+
+  .logo-transparent {
+  height: 10vw;
+  width: auto;
+  float: left;
+  }
+
+  .nav-bar {
+  display:flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  align-items: center;
+  }
+  
+  .main-nav li {
+  list-style-type: none;
+  display: inline-block;
+  padding: 15px 15px;
+  }
+  
+  a {
+  text-decoration: none;
+  color: #FC5424;
+  } 
+
+  a:hover {
+  color: #B7448E;
+  }
+  
+  /* ----------------------------------------------- */
+  /*                  MAIN NEWS                      */
+  /* ----------------------------------------------- */
+ 	  
+    .main-news-container {
+    display: flex;
+    flex-direction: row;
+    margin-top: 3%;
+    padding: 8%;
+    background-color: #f4f4f4;
+  }
+    
+    .main-news-img {
+    max-width: 100%;
+    height: auto;
+    }
+    
+    .main-news-contents {
+    margin-left: 3%;
+    } 
+
+    .main-subtitle {
+      margin-bottom: 2%;
+      color: #FC774B;
+    }
+
+    .read-more-link {
+      font-size: 80%;
+    }
+  
+  /* ----------------------------------------------- */
+  /*                  FEATURED NEWS                  */
+  /* ----------------------------------------------- */
+ 	  
+  .featured-news-container {
+  margin-top: 3%;
+  width: 100%;
+  background-color: #f4f4f4;
+  padding: 8%;
+  }
+  
+  .grid {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  grid-gap: 30px;
+  align-items: stretch;
```

Comment:
That's a good question! If you test other values for the align-items property (like center or end), you'll notice the sizes of the pictures don't match anymore. So to prevent this, I made all pictures "stretch" to the standard size of their respective grid fields :)

Reply:
Ah nice! 👌